content/overview/installation/_index.en.md

```diff
@@ -42,6 +42,11 @@ You can find our packagecloud.io page [here](https://packagecloud.io/wasmcloud/c
    {{% /tab %}}
 {{% /tabs %}}
 
+⚠️ **M1 Mac Users** - The default engine for wasmCloud, `wasm3`, does not currently support M1 Macs. You can still use `wasmcloud` and `wash` by installing with the `wasmtime` feature flag, shown by the following command:

```

Comment:
"default engine" may not mean much to new users. Maybe change it to the "default WebAssembly engine"?